app/cart/Cart.tsx

```diff
@@ -9,29 +9,37 @@ import { ProductInCartDTO } from "../../types";
 import axios from "axios";
 
 export default function Cart() {
+  // te dwa stany być może powinny byś złączone w jeden stand
```

Comment:
to jest tylko żeby chwilkę na ten temat pomyśleć - lepiej zostaw jak jest teraz :)